pallets/msa/src/lib.rs

```diff
@@ -498,9 +498,29 @@ impl<T: Config> Pallet<T> {
 impl<T: Config> AccountProvider for Pallet<T> {
 	type AccountId = T::AccountId;
 	type BlockNumber = T::BlockNumber;
+
+	#[cfg(not(feature = "runtime-benchmarks"))]
```

Comment:
#h4x,  but I guess I wish I'd thought of it lol

Can we make an issue or something to make sure we don't leave  it this way forever?

Reply:
Sure, I created a new issue for it. https://github.com/LibertyDSNP/mrc/issues/154